recipe-server/client/control_new/utils/apiFetch.js

```diff
@@ -62,9 +62,9 @@ export default async function apiFetch(url, options = {}) {
     throw new ApiError(message, data);
   }
 
-  if (response.status !== 204) {
+  try {
     return response.json();
```

Comment:
I think a slightly cleaner way to put that would be 

```js
try {
  return await response.json();
} catch(e) {
  return null;
}
```

This has the `return await` that we've been talking about in linting lately, but in this case it is actually needed (for the reason @andymikulski pointed out), and the linter understands this and will let it go.